### PR TITLE
Add visa pages for non-UK trainees

### PR DESCRIPTION
**Pull request by ethax-ross**
Add visas page for non-UK trainees

### CODE REVIEW

File: config/redirects.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ redirects:
   # International content
   "/iqts": "/non-uk-teachers/international-qualified-teacher-status"
   "/funding-and-support/if-you-come-from-outside-england": "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"
+  "/non-uk-teachers/fees-and-funding-for-non-UK-trainees": "/non-uk-teachers/fees-and-funding-for-non-uk-trainees"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:29]: Looks to me like this redirect is already in the redirects file @laurahermionetennant92 

image

Comment (laurahermionetennant92) [reply config/redirects.yml:29]: That is above my Github pay grade @gemmadallmandfe! Do I need to take an action?

File: app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md [hunk 2]
```diff
@@ -162,66 +167,102 @@ If you are not training to teach in these subjects, any support you get will dep
...
+Salaried courses include:
 
-#### Choosing a subject to teach and preparing to apply
+* School Direct salaried teacher training programmes
+* the Teach First Leadership Development programme
+* postgraduate teaching apprenticeships 
+ 
+#### Choosing a subject to teach 
 
 The subject you apply to teach must usually be the same, or closely related to, the subject you studied at university. (For example, engineering graduates can apply to train to teach physics, but computer science graduates are unlikely to be accepted to train to teach geography.)
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:194]:
```suggestion
The subject you apply to teach must usually be the same as, or closely related to, the subject you studied at university. (For example, engineering graduates can apply to train to teach physics, but computer science graduates are unlikely to be accepted to train to teach geography.) 
```

Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:194]: Argh good spot!!!

File: app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md [hunk 3]
```diff
@@ -77,7 +78,7 @@ Teacher training in England is practical, hands-on and supportive. As an interna
 
 When you have successfully completed your training, you will be awarded [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts), a professional qualification valued globally.
 
-QTS will allow you to teach in all schools in England. Some courses combine QTS with a [postgraduate certificate in education (PGCE) or postgraduate diploma in education (PGDE)](/what-is-a-pgce). These are optional: you do not need a PGCE or PGDE to teach in England.
+QTS will allow you to teach in all schools in England.
```
[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:81]: Ah interesting – why most? 
QTS is a legal requirement in some schools, but not all schools, so if you DON'T have QTS you will be able to teach in 'some' schools but if you have QTS you are legally qualified to teach in 'all' schools.

[the remark this answers is not in the document]
Comment (philspray) [reply app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:81]: To account for schools who can set their own requirements, e.g. academies and private schools.

[the remark this answers is not in the document]
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md:81]: OK – I've tweaked it so it says 'apply for jobs' so we're not implying they will necessarily get a job...

**Review by philspray — APPROVED** [review]

**Comment by laurahermionetennant92** [comment]
@philspray @EllieNodder can we merge and deploy?

**Pull request merged**